src/figuresClasses.ts

```diff
@@ -1,19 +1,109 @@
 export interface Figure {
-
+  form: 'triangle' | 'circle' | 'rectangle',
```

Comment:
create a separate type for it and use everywhere you need (the same for color)
```suggestion
  shape: Shape,
```